serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -119,11 +119,13 @@ mvn clean package
 Knative CLI::
 +
 --
-.Build your project and generate a local image called `quay.io/other-user/my-project:1.0.1`
+.Build your project and generate a local image called `dev.local/serverless-workflow-hello-world:latest`
```

Comment:
doesn't the image needs to contain the full registry address? registry/context/image_name:tag?

Reply:
Actually, to work locally (without having to push the image) we should include only `dev.local/image_name:tag`